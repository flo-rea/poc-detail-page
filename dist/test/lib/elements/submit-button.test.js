"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const submit_button_1 = require('../../../src/lib/view/elements/submit-button');
jest.mock('react-dom');
describe('SubmitButton', () => {
    it('has correct default options', () => {
        const component = renderer.create(React.createElement(submit_button_1.default, {id: "button"}, "submit-me!"));
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=submit-button.test.js.map