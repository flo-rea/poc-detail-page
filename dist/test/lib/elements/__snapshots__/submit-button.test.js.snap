exports[`SubmitButton has correct default options 1`] = `
<span>
  <button
    id="button"
    type="submit">
    submit-me!
  </button>
</span>
`;
